packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,90 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs, useId } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box, linkStyles } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export function useLinkDescriptionId(idProp?: string) {
+	const autoId = useId(idProp);
+	const sanitizeId = autoId?.replace(/[^a-zA-Z0-9-_]/g, '_'); //sanitize id to have valid html test pass
+	return `link-desc-${sanitizeId}`;
+}
+
+export type FeatureLinkListItemProps = LinkProps & {
+	label: ReactNode;
+	description?: ReactNode;
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkListItem = ({
+	label,
+	description,
+	background = 'body',
+	...props
+}: FeatureLinkListItemProps) => {
+	const TextComponent = props.target == '_blank' ? TextLinkExternal : TextLink;
+	const descriptionId = useLinkDescriptionId(props.id);
+
+	return (
+		<Box
+			as="li"
+			borderBottom
+			background={background}
+			css={[
+				{ ...linkStyles, textDecoration: 'none' },
+				{
+					'&:hover': {
+						backgroundColor: hoverColorMap[background],
+					},
+				},
+			]}
```

Comment:
Could just make this all in 1 object with no array since you're spreading anyway
```suggestion
			css={{
				...linkStyles,
				textDecoration: 'none',
				'&:hover': {
					backgroundColor: hoverColorMap[background],
				},
			}}
```